src/components/implementation/tests/unit_pongshmem/pong.c

```diff
@@ -0,0 +1,29 @@
+#include <cos_kernel_api.h>
+#include <cos_types.h>
+#include <ps.h>
+#include <capmgr.h>
+#include <memmgr.h>
+#include <pongshmem.h>
+
+int
```

Comment:
Can get rid of the main. See the weak version in cos_component to understand why.